trace/obfuscation_config.yaml

```diff
@@ -1,9 +1,157 @@
+  - '[-]{5}BEGIN RSA PRIVATE KEY[-]{5}([\s\S]{128,}?)[-]{5}END RSA PRIVATE KEY[-]{5}'
+  - (?i)\bduffel_test_[a-zA-Z0-9_-]{43}\b
+  - (?i)\br8_[0-9a-z-_]{37}\b
+  - (?i)\bhf_[0-9a-z]{34}\b
+  - \b[a-f0-9]{8}:[a-f0-9]{8}\b
+  - \bakaa[0-9a-z-]{15,1000}\b
+  - (?i)\bghr_[0-9a-zA-Z]{36}\b
+  - (?i)\bshippo_live_[a-fA-F0-9]{40}\b
+  - \bglptt-[0-9a-f]{40}\b
+  - \bdapi([a-hA-H0-9]{32})\b
+  - \bpscale_app_secret_[a-zA-Z0-9=\-_\.]{43}\b
+  - Bearer xox[os]-\d+-\d+-\d+-[a-fA-F\d]+
+  - \bdt0c01\.[a-zA-Z0-9]{24}\.[a-zA-Z0-9]{64}\b
+  - \b(glc_[A-Za-z0-9+\/]{32,400}={0,2})\b
+  - (?i)\brubygems_[a-f0-9]{48}\b
+  - (?i)\bCCIPAT_[0-9a-z]{22}_[0-9a-z]{40}\b
+  - \bNRII-[a-zA-Z0-9-]{32}\b
+  - Bearer xoxb-[0-9]{10,13}\-[0-9]{10,13}[a-zA-Z0-9-]*
+  - (?i)\bghp_[A-Z0-9]{36}\b
+  - \bakab-[a-zA-Z0-9]{16}-[a-zA-Z0-9]{16}\b
+  - (?i)\bgh[us]_[0-9a-zA-Z]{36}\b
+  - \bGR1348941[0-9a-zA-Z\-\_]{20}\b
+  - \bdp\.ct\.[a-zA-Z0-9]{40,44}\b
+  - \bapi_org_[a-zA-Z]{34}\b
+  - \beyJrIjoi[A-Za-z0-9]{70,400}={0,2}\b
+  - \btk-us-[a-zA-Z0-9-_]{48}\b
+  - key"\s*:\s*"X-Auth-Key"\s*,\s*"value"\s*:\s*"([a-f0-9]{37})
+  - \bAGE-SECRET-KEY-1[QPZRY9X8GF2TVDW0S3JN54KHCE6MUA7L]{58}\b
+  - \bsu[a-zA-Z0-9]{12}\b
+  - (?i)\bBasic [A-Z0-9+/]{8,1000}[=]{0,2}
+  - '[-]{5}BEGIN DSA PRIVATE KEY[-]{5}([\s\S]{128,}?)[-]{5}END DSA PRIVATE KEY[-]{5}'
+  - \bdG9rO[0-9a-zA-Z]{54}\=
+  - \bphc_[a-zA-Z0-9_]{43}\b
+  - \bBearer [A-Za-z0-9\-._~+/]{8,1000}[=]{0,2}
+  - (?i)\bNRAK-[0-9a-z-_]{27}\b
+  - (?i)\bgho_[0-9a-zA-Z]{36}\b
+  - (?i)\bpul-[a-fA-F0-9]{40}\b
+  - (?i)\bhttps:\/\/chat\.twilio\.com\/v2\/Services\/[a-zA-Z0-9]{32}\b
+  - \bpub-c-[0-9a-z]{8}-[0-9a-z]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}\b
+  - \baio\_[a-zA-Z0-9]{28}\b
+  - \b(live|test)_[a-f0-9]{35}\b
+  - \bpk\.[a-zA-Z0-9]{60,70}\.[a-zA-Z0-9]{22}\b
+  - '[-]{5}BEGIN PGP PRIVATE KEY BLOCK[-]{5}([\s\S]{128,}?)[-]{5}END PGP PRIVATE KEY BLOCK[-]{5}'
+  - \bsk_[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}\b
+  - (?i)\bdo[por]_v1_[a-f0-9]{64}\b
+  - \bey[a-zA-Z0-9]{17,512}\.ey[a-zA-Z0-9/_-]{17,512}\.[a-zA-Z0-9/_-]{17,512}={0,2}\b
+  - \bLTAI[a-zA-Z0-9]{20}\b
+  - \brdme_[a-zA-Z0-9]{70}\b
+  - \bsecret_[0-9a-zA-Z-_]{43}\b
+  - (?i)\bpk_[0-9]{7,8}_[0-9a-z]{32}\b
+  - key"\s*:\s*"Authorization"\s*,\s*"value"\s*:\s*"(Bearer [0-9]{15,25}-[a-zA-Z0-9]{20,40})
+  - \bpnu_[a-zA-Z0-9]{36}\b
+  - \bsub-c-[0-9a-z]{8}-[a-z]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12}\b
+  - \bfio-u-[a-zA-Z0-9\-_=]{64}\b
+  - (?i)\brk_live_[0-9a-z]{24}\b
+  - \bion_[a-zA-Z0-9]{42}\b
+  - \bkey[a-zA-Z0-9]{14}\b
+  - https:\/\/www\.google\.com\/calendar\/embed\?src=[A-Za-z0-9%\@&;=\-_\.\/]+
+  - \bpdct\.1\.1\.[0-9A-Z]{16}\.[0-9a-z]{16}\.[0-9a-z]{40}\b
+  - \bYC[a-zA-Z0-9_\-]{38}\b
+  - \bBBFF-[0-9a-zA-Z]{30}\b
+  - (?i)\bpscale_tkn_[a-zA-Z0-9\-_\.]{43}\b
+  - \bEZTK[a-zA-Z0-9]{54}\b
+  - \bapify\_api\_[a-zA-Z-0-9]{36}\b
+  - \bEAACEdEose0cBA[0-9A-Za-z]{5,1000}\b
+  - (?i)[0-9a-z+\/]{15,1000}
```

Comment:
This one also lacks context, we cannot remove all 15-character alphanumeric strings.

This would partially match any url included in a response.